tests/test_java_files/send_broadcast_receiver_permission.java

```diff
@@ -0,0 +1,418 @@
+/*
```

Comment:
Would it be possible to break this up into individual java files and run the test on each one of them? It's non obvious how many errors are in this file

Reply:
I've modified the java file so its a lot easier to read.